docs/src/main/paradox/docs/getting-started/running-nexus/configuration/index.md

```diff
@@ -64,34 +62,36 @@ say the first time, they query the project listing endpoints.
+* The current username where only non-diacritic alphabetic characters (`[a-zA-Z]`), numbers, dashes and underscores will be preserved.
 * This resulting string is then truncated to 64 characters if needed.
 
-This feature can be turned on via the flag `app.projects.automatic-provisioning.enabled`.
+This feature can be turned on via the flag `app.automatic-provisioning.enabled`.
 
-@link:[This section](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/app/src/main/resources/app.conf#L223){ open=new } of the configuration defines the project provisioning configuration.
+@link:[The `automatic-provisioning` section](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/app/src/main/resources/app.conf#L197){ open=new } of the configuration defines the project provisioning configuration.
 
 ## Encryption configuration
 
 Nexus Delta uses symmetric encryption to secure sensitive data information (tokens and passwords).
 
-@link:[This section](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/app/src/main/resources/app.conf#L298){ open=new } of the configuration defines the encryption configuration.
+@link:[The `encryption` section](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/app/src/main/resources/app.conf#L276){ open=new } of the configuration defines the encryption configuration.
 
 ## Fusion configuration
 
 When fetching a resource, Nexus Delta allows to return a redirection to its representation in Fusion by providing `text/html` in the `Accept` header.
 
-@link:[This section](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/app/src/main/resources/app.conf#L85){ open=new } of the configuration defines the fusion configuration.
+@link:[The `fusion` section](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/app/src/main/resources/app.conf#L40){ open=new } of the configuration defines the fusion configuration.
 
 ## Plugins configuration
 
 Since 1.5.0, Nexus Delta supports plugins. Jar files present inside the local directory defined by the `DELTA_PLUGINS` environment variable are loaded as plugins into the Delta service. 
 
 Each plugin configuration is rooted under `plugins.{plugin_name}`. All plugins have a `plugins.{plugin_name}.priority` configuration flag used to determine the order in which the routes are handled in case of collisions. 
 
+For more information about plugins, please refer to the @ref:[Plugins page](../../../delta/plugins/index.md).
+
 ### Elasticsearch views plugin configuration
 
-The elasticsearch plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/plugins/elasticsearch/src/main/resources/elasticsearch.conf){ open=new }. 
+The elasticsearch plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/$git.branch$/delta/plugins/elasticsearch/src/main/resources/elasticsearch.conf){ open=new }.
```

Comment:
It would be nice to add a mention about the indexing errors.
Same for the slow queries for Blazegraph

Reply:
I'm going to remove it from the doc and it will be fixed by closing #3839